demo/index.html

```diff
@@ -1,29 +1,37 @@
 <!doctype html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
-  <title>vaadin-radio-button Examples</title>
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1"/>
+    <title>vaadin-radio-button Examples</title>
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 
-  <link rel="import" href="../../polymer/polymer.html">
-  <link rel="import" href="../../vaadin-demo-helpers/vaadin-component-demo.html">
-  <link rel="import" href="../../vaadin-demo-helpers/vaadin-demo-snippet.html">
-  <script src="../../vaadin-demo-helpers/vaadin-demo-ready-event-emitter.js"></script>
+    <link rel="import" href="../../polymer/polymer.html">
+    <link rel="import" href="../../vaadin-demo-helpers/vaadin-component-demo.html">
+    <link rel="import" href="../../vaadin-demo-helpers/vaadin-demo-snippet.html">
+    <script src="../../vaadin-demo-helpers/vaadin-demo-ready-event-emitter.js"></script>
+    <script src="radio-button-demo.js"></script>
 
-  <script src="radio-button-demo.js"></script>
+    <link rel="import" href="../../vaadin-button/vaadin-button.html">
+    <link rel="import" href="../vaadin-radio-button.html">
+    <link rel="import" href="../vaadin-radio-group.html">
 
-  <link rel="import" href="../../iron-form/iron-form.html">
-  <link rel="import" href="../../vaadin-button/vaadin-button.html">
-  <link rel="import" href="../vaadin-radio-button.html">
-  <link rel="import" href="../vaadin-radio-group.html">
+    <custom-style>
+      <style include="vaadin-component-demo-shared-styles"></style>
+    </custom-style>
+  </head>
 
-  <custom-style>
-    <style include="vaadin-component-demo-shared-styles"></style>
-  </custom-style>
-</head>
-
-<body>
-  <vaadin-component-demo config-src="demos.json"></vaadin-component-demo>
-</body>
+  <body>
+    <vaadin-component-demo config-src="demos.json"></vaadin-component-demo>
+  </body>
 </html>
+
+<dom-module id="custom-radio-button-icon-color" theme-for="vaadin-radio-button">
```

Comment:
Can you please move `dom-module` tag inside `body`? 

It will be rendered by browser there anyways. Nothing should be outside `<html>`

Reply:
Done.